qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -115,18 +128,54 @@ def __init__(
         self._properties = None
         self._defaults = None
 
+    @classmethod
+    def _default_options(cls) -> Options:
+        """Default runtime options."""
+        return Options(shots=1024, memory=False,
+                       qubit_lo_freq=None, meas_lo_freq=None,
+                       schedule_los=None,
+                       meas_level=MeasLevel.CLASSIFIED,
+                       meas_return=MeasReturnType.AVERAGE,
+                       memory_slot_size=100,
+                       rep_time=None, rep_delay=None,
+                       init_qubits=True)
+
+    @deprecate_arguments({'qobj': 'circuits'})
     def run(
             self,
-            qobj: Union[QasmQobj, PulseQobj],
+            circuits: Union[QasmQobj, PulseQobj, QuantumCircuit, Schedule,
+                            List[Union[QuantumCircuit, Schedule]]],
             job_name: Optional[str] = None,
             job_share_level: Optional[str] = None,
             job_tags: Optional[List[str]] = None,
-            validate_qobj: bool = False
+            validate_qobj: bool = False,
+            header: Optional[Dict] = None,
+            shots: Optional[int] = None,
+            memory: Optional[bool] = None,
+            qubit_lo_freq: Optional[List[int]] = None,
+            meas_lo_freq: Optional[List[int]] = None,
+            schedule_los: Optional[Union[List[Union[Dict[PulseChannel, float], LoConfig]],
+                                         Union[Dict[PulseChannel, float], LoConfig]]] = None,
+            meas_level: Optional[Union[int, MeasLevel]] = None,
+            meas_return: Optional[Union[str, MeasReturnType]] = None,
+            memory_slot_size: Optional[int] = None,
+            rep_time: Optional[int] = None,
+            rep_delay: Optional[float] = None,
+            init_qubits: Optional[bool] = None,
+            **run_config: Dict
     ) -> IBMQJob:
-        """Run a Qobj asynchronously.
+        """Run on the backend.
+
+        If a keyword specified here is also present in the ``options`` attribute/object,
+        the value specified here will be used for this run.
 
         Args:
-            qobj: The Qobj to be executed.
+            circuits: An individual or a
+                list of :class:`~qiskit.circuits.QuantumCircuit` or
+                :class:`~qiskit.pulse.Schedule` objects to run on the backend.
+                A :class:`~qiskit.qobj.QasmQobj` or a
```

Comment:
Do we want to mention that Qobj support is deprecated here?

Reply:
It does mention Qobj is deprecated:
```
A :class:`~qiskit.qobj.QasmQobj` or a :class:`~qiskit.qobj.PulseQobj` object is also supported but
                is deprecated.
```
The API doc should be easier to read without all the sphinx tags.

Reply:
Ah yep, sorry I missed it in there.